.kitchen.yml

```diff
@@ -9,9 +9,15 @@ provisioner:
   chef_omnibus_install_options: -d /tmp/vagrant-cache/vagrant_omnibus
 
 platforms:
-  - name: centos-5.11
-  - name: centos-6.6
-  - name: centos-7.0
+<% %w(5.11 6.6 7.0).each do |version| %>
+  - name: centos-<%= version %>
+    run_list:
+      - recipe[yum]
+    attributes:
+      yum:
+        main:
+          keepcache: true
+<% end %>
```

Comment:
All good except `inherits: "magento"` in the `stack` suite on chef 12.6.0 on centos 6.7 or 7.2 is not liked.
Without `type: magento` it fails to write out the cron entry, logrotate file or the local.xml file and thus the verify step fails.

Kitchen-Vagrant is using the latest chef (12.6.0) but the docker driver is hardcoded to use 11.18.12 to speed things up. The stack suite is fine on chef 11.18.12, which is why I didn't catch this the other day.

Would you like:
1) multiple chef versions being run (like config-driven-helper)
2) vagrant to be hardcoded to 11.18.12
3) docker to install the latest chef version
4) something else :)